scripts/npm-postversion-for-bower.sh

```diff
@@ -0,0 +1,16 @@
+#!/usr/bin/env bash
```

Comment:
@misteroneill I don't see that branch on https://github.com/videojs/generator-videojs-plugin, is there somewhere else I should be looking?